### DISCUSSION TITLE
Richtlijnen: Feedback op richtlijnen NLDS

### DISCUSSION THREAD

**Discussion by rianrietveld** [nl-design-system/documentatie#473] [Richtlijnen NLDS]
Het NL Design System schrijft [richtlijnen ](https://nldesignsystem.nl/richtlijnen/.
Richtlijnen voor toegankelijkheid en gebruikersvriendelijkheid, dus niet alleen gebaseerd op de officiële WCAG succescriteria, maar ook gebaseerd op gebruikersonderzoek. 

Een combinatie van WCAG en resultaten van (gebruikers)onderzoek, in leesbaar Nederlands, voor content-managers, designers en developers. Met uitleg, code en design-voorbeelden (doen en niet doen) en links naar WCAG of onderzoeksresultaten. 

Graag zouden we je mening willen vragen. Zijn we op de goede weg, wat vind je van de opzet en ben je eens met wat er staat. Ons doel is dat deze richtlijnen breed gedragen gaan worden, binnen overheid en hopelijk ook daarbuiten. Het worden handvatten hoe het beste een formulier samen te stellen. 

Er komt in de introductiepagina nog een verhaal over het onderscheid tussen de wettelijk verplichte WCAG succescriteria en de niet wettelijk verplichte richtlijnen gebaseerd op onderzoek.

Heb je aanvullingen en/of feedback, voeg een comment toe aan dit issue.

Alle feedback is welkom 🌷

**Comment by rianrietveld** [comment]
Feedback via Slack van @Rozerinay 
[...] 

- Het is handig dat bovenaan een opsomming staat van de belangrijke punten, hoewel ik dit miste bij ‘Descriptions’. Het zou nog handiger zijn als de punten in de opsomming direct gelinkt zijn naar de bijbehorende kop. Dit zou een ankernavigatie mogelijk maken, waardoor we sneller kunnen navigeren naar het specifieke punt dat we belangrijk vinden, in plaats van te scrollen en te zoeken.
- Het zou misschien leuk zijn om de opsomming in de toekomst te laten fungeren als een checklist die we kunnen afvinken. Dit kan een praktische toevoeging zijn.
- De voorbeelden van ‘Doen’ en ‘Niet doen’ zijn erg nuttig. Ze bieden directe praktijkinzichten over hoe we bepaalde zaken wel of niet moeten aanpakken.
- De taal is zeer begrijpelijk, en ik waardeer het gebruik van korte zinnen.

[...]

**Reply by rianrietveld** [reply]
@Rozerinay 
De opsomming is aan de [descriptions](https://nldesignsystem.nl/richtlijnen/formulieren/alle-richtlijnen/descriptions) toegevoegd, bedank voor het opmerken :-)
Voor het [linken naar de kopjes](https://github.com/nl-design-system/documentatie/issues/606) heb ik een issue aangemaakt, dat gaan we maken.

**Comment by rianrietveld** [comment]
Feedback via Slack van Diede

Hi, ik heb een labels pagina door genomen. Ik vind het fijn om te lezen waarom dingen belangrijk zijn en ook de uitleg terug te lezen van voor welke doelgroepen je het doet. Ik wist bijvoorbeeld niet dat labels ook voor voice commands worden gebruikt.
Eerst overkoepelende verbetering die ik mogelijk zie is dat het fijn zou zijn als de titels anchors zouden zijn zodat je een specifieke tip kan delen met collega's, vaak wil ik collega's niet doorverwijzen naar de zo'n hele tips pagina maar naar een specifiek probleem wat we proberen te tackelen.

@hidde heeft hier een issue voor aangemaakt: https://github.com/nl-design-system/documentatie/issues/427

**Reply by rianrietveld** [reply]
Is aangepast.

**Comment by AlineNap** [comment]
2 punten voor nu:
- bij description staat een don't beschreven over foutmelding onder het invoerveld. Is hier een onderbouwing voor? En wellicht had ik die eerder verwacht bij voorkom fouten of bij een andere kop. 
- Een checkbox en radio button heeft ook een label en description, wellicht goed om dit te benoemen en/of onderscheid in te maken? 

Thanks voor dit uitschrijven, heel waardevol!

**Comment by Rozerinay** [comment]
Richtlijnen 'Niet doen' zetten we neer in een blok. Echter, bij het [voorbeeld dat je geen link in het label moet plaatsen](https://arc.net/l/quote/zvvoogsv), is er gebruikgemaakt van een ander type blok (met een rode border).

Moet dat voorbeeld niet ook worden opgenomen in het 'Niet doen' richtlijnen blok?

**Comment by Rozerinay** [comment]
Bij [Plaatsing van een button binnen een formulier](https://arc.net/l/quote/lkrpdrgu) zijn er 2 afbeeldingen bijgevoegd.

Bij deze voorbeelden zou het handig zijn om te kunnen zien welke wordt aanbevolen (de eerste afbeelding) en welke wordt afgeraden (de tweede afbeelding).

Dit geldt ook voor het voorbeeld dat is toegevoegd bij [Duidelijk buttontekst die beschrijft wat de button doet](https://arc.net/l/quote/bwoddsuf).

**Comment by Rozerinay** [comment]
Bij [Vermeld duidelijk of een veld verplicht is](https://arc.net/l/quote/kncdemff) is er in het eerste bullet point het woord "optioneel" genoemd. Dit woord is [niet b1](https://ishetb1.nl/search?word=optioneel). Een alternatief hiervoor is "niet verplicht".
